crates/uv-configuration/src/concurrency.rs

```diff
@@ -26,8 +31,8 @@ impl Concurrency {
     // The default concurrent downloads limit.
     pub const DEFAULT_DOWNLOADS: usize = 50;
 
-    // The default concurrent builds limit.
-    pub fn default_builds() -> usize {
+    // The default concurrent builds and install limit.
+    pub fn threads() -> usize {
```

Comment:
Just confirming that this is the same as the Rayon default?

Reply:
Yes, https://docs.rs/rayon-core/1.12.1/src/rayon_core/lib.rs.html#473.